tests/unit/utils_test.py

```diff
@@ -587,6 +587,9 @@ def test_with_no_container_port(self):
     def test_split_port_empty_string(self):
         self.assertRaises(ValueError, lambda: split_port(""))
 
+    def test_split_port_non_string(self):
+        assert split_port(1243) == (['1243'], None)
+
```

Comment:
No commit reviews / reviews then commit? This test doesn't look like any of the others.